haproxy-healthcheck/compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.7"
+services:
+  mock-couchdb:
+    image: mockserver/mockserver:5.15.0
+    environment:
+      MOCKSERVER_SERVER_PORT: 5984
+      # MOCKSERVER_LOG_LEVEL: WARN
+      MOCKSERVER_INITIALIZATION_JSON_PATH: /config/initializerJson.json
+    volumes:
+      - ./mock-config/:/config/
+
+  haproxy-healthcheck:
+    build: .
+    environment:
+      COUCHDB_SERVERS: mock-couchdb
+      COUCHDB_USER: someuser
+      COUCHDB_PASSWORD: insecure_pw
+      APP_LOGLEVEL: DEBUG
```

Comment:
Can `APP_LOGLEVEL` be renamed to `HEALTHCHECK_LOG_LEVEL` ? `APP_LOGLEVEL` is really generic and we use `COUCHDB_LOG_LEVEL` for Couch for example. 
Also, can you please add this to the docker compose templates and to the upgrade service (separate PR)?

Reply:
Sure, PR updated

Reply:
Added PR for upgrade service. Since it's a bit of a niche configuration, is it overkill to add it to the docker compose template? (I assume you mean `scripts/docker-helper-4.x/cht-docker-compose.sh`?)